src/style.css

```
@@ -0,0 +1,151 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap");
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.article {
+  position: relative;
+  width: 160px;
+  height: 134px;
+  left: calc(50% - 160px / 2);
+
+}
+
+.article-img {
+  display: block;
+  margin: 0 auto 40px;
```

```suggestion
  margin-bottom: 40px;
``` Думаю хватит только нижнего отступа, потому что основные падинги ты задал в .container